draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -87,17 +89,43 @@ With QUIC, the equivalent transport headers are encrypted and passive packet
 loss observation is not possible, as described in {{TRANSPORT-ENCRYPT}}.
 
 Measuring TCP loss between similar endpoints cannot be relied upon to
-evaluate QUIC loss.  QUIC could be routed by the network differently and
+evaluate QUIC loss. QUIC could be routed by the network differently and
 the fraction of Internet traffic delivered using QUIC is increasing every
 year.  It is imperative to measure packet loss experienced by QUIC users
 directly.
 
-Since explicit path signals are preferred by {{!RFC8558}}, two explicit loss
+## On-path loss observation
+
+There are three sources of loss that network operators need to observe
+to guarantee high QoS : 
+
+* _upstream loss_ - loss between the sender and the observation point
+  ({{upstreamloss}})
+
+* _downstream loss_ - loss between the observation point and the destination
+  ({{downstreamloss}})
+
+* _observer loss_ - loss by the observer itself that does not cause downstream
+  loss ({{observerloss}})
+
+The upstream and downstream loss together constitute _end-to-end loss_
+({{endtoendloss}}).
+
+## How on-path loss observation can be achieved in QUIC v1
+
+{{LOSSBITS}} presents the generic technique. Two explicit loss
 bits in the clear portion of short headers are used to signal packet loss to
-on-path network devices.
+on-path network devices. This approach follows the recommandations of 
+{{!RFC8558}} where it is mentionned that explicit path signals are preferred. 
 
-This draft adapts the general technique described in {{LOSSBITS}} for QUIC using
-reserved bits in QUIC v1 short header.
+The current document adapts the technique proposed in {{LOSSBITS}} for QUIC
+by using reserved bits in QUIC v1 short header the "sQuare signal" bit (Q)
+and the "Loss event" bit (L).
+
+While the exploitation of only Q can help in measuring the _upstream loss_ 
+and only L can help in measuring the _end-to-end loss_, both are conjointly 
+required to detect and measure the other types of losses (_downstream loss_ 
```

Comment:
Is this sentence missing a word?  Perhaps

> [...] by using reserved bits **as** the "sQuare signal" bit...

Reply:
Thanks - corrected in the last commit.